panel/src/components/Forms/Fieldset.vue

```diff
@@ -3,7 +3,7 @@
 		<k-grid>
 			<template v-for="(field, fieldName) in fields">
 				<k-column
-					v-if="$helper.field.isVisible(field, value)"
+					v-if="$helper.field.isVisible(field, context)"
```

Comment:
Is `context` still a single value? It sounds like ti would be now multiple fieldsets/many values

Reply:
`value` was the "value of the fieldset", in other words this always contained "many values". 
`context` however now combines the values of many fieldsets (it includes all parent fieldsets, "flattened" to a single object). This is how we give the `isVisible` helper access to fields in parent fieldsets. 

The `isVisible` interface doesn't change.

Reply:
What does happen when layers contain fields with the same name?

Reply:
```js
return {
	...this.injectedContext,
	...this.value
};
```

`this.value` shadows variables that might have the same name. This is how it's normally handled in programming languages which support scoping. Example in javascript:

```js
const variable = 'outer';
{
  const variable = 'inner';
  console.log(variable); // -> 'inner'
}
console.log(variable); // -> 'outer'
```